services/plotter/utils/wrapper.py

```diff
@@ -0,0 +1,45 @@
+from io import BytesIO
+from typing import List
+
+import matplotlib
+import matplotlib.pyplot as plt
+from matplotlib.figure import Figure
+from matplotlib.axes import Axes
+
+from models.payload import PayloadModel
+from models.image import FigureModel, LayoutModel, GraphModel, PlotModel
+
+
+matplotlib.use("agg")
+
+
+def build_image(payload: PayloadModel):
```

Comment:
for consistency, it should have return type too

Reply:
You're right, but for this PR I've focused mainly on the service functionality, not on service features. Later the service will get some refactorization with the definite data model definition.